### PR TITLE
Update wishlist items

### PR DESCRIPTION
**Pull request by Harshbansal1407**
This PR makes following updates:
- Added `update_items` in `service/routes.py`
- Added test cases in `tests/test_routes.py`
- Added test cases in `tests/test_wishlist_item.py`

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -230,3 +230,41 @@ def get_addresses(wishlist_id, id):
...
+        abort(
+            status.HTTP_404_NOT_FOUND,
+            f"Wishlist with id '{wishlist_id}' could not be found.",
+        )
+    # See if the item exists and abort if it doesn't
+    item = WishListItem.find(item_id)
+    if not item:
+        abort(
+            status.HTTP_404_NOT_FOUND,
+            f"Item with id '{item_id}' could not be found in the Wishlist with id '{wishlist_id}'.",
```
Comment (arunimamitra) [inline service/routes.py:261]: There should be an additional check to see whether item id provided belongs to the specified wishlist or not. Say I make a PUT API call for {BASE_URL}/wishlists/1/items/2 but item id=2 belongs to wishlist id=2 and not wishlist id=1, then I should receive an error saying item id=2 not found for wishlist 1. You can check the read API call for reference

**Review by arunimamitra — CHANGES_REQUESTED** [review]

File: service/routes.py (hunk 1 above)
Comment (Harshbansal1407) [reply service/routes.py:261]: I have updated the test case in the code.

**Review by Animesh-Ramesh — COMMENTED** [review]
Minor naming changes

**Review by arunimamitra — APPROVED** [review]

File: tests/test_routes.py [hunk 2]
```diff
@@ -399,3 +426,102 @@ def test_delete_wishlist_item(self):
...
+        resp = self.client.put(
+            f"{BASE_URL}/{wishlist.id}/items/{item_id}",
+            json=data,
+            content_type="application/json",
+        )
+        
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_update_item_with_incorrect_wishlist_id(self):
+        """It should not be able to update an existent item if wishlist_id provided is wrong"""
```
[the remark this answers is not in the document]
Comment (Harshbansal1407) [reply tests/test_routes.py:501]: They are for different cases. This has been explained by @arunimamitra in the [resolved conversation](https://github.com/CSCI-GA-2820-SP24-003/wishlists/pull/44/files/c7c5393ed9136525d5ca5733fa367d8c5bd00415)

**Pull request merged**